Source/WebCore/animation/StyleOriginatedTimelinesController.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2025 Apple Inc. All rights reserved.
```

Comment:
Shouldn't this start at 2020? Prolly same for the other file.

Reply:
You're right, I'll fix it.